SCClassLibrary/Common/Core/Object.sc

```diff
@@ -169,6 +183,14 @@ Object  {
 	valueEnvir { ^this }
 	valueArrayEnvir { ^this }
 
+	valueWith {|argumentsArray, keywordArgumentPairs|
+		^this.valueArray(
+			this.class
+                .findRespondingMethodFor(\value)
```

Comment:
just indentation

Reply:
SCIDE does not let me insert the tab on the new line. I've had to use another program to follow the style guide.